### DISCUSSION TITLE
is the HSM integration available in the community version?

### DISCUSSION THREAD

**Discussion by bsjaramillo** [Keyfactor/ejbca-ce#319] [General]
Or is an enterprise feature ?

**Comment by primetomas** [comment]
Yes, basic HSM integration (using the Java PKCS#11 provider) is in Community. Check here, https://doc.primekey.com/ejbca/ejbca-integration/hardware-security-modules-hsm
The things that are Enterprise are clearly labelled with an "Enterprise" label.

**Reply by bsjaramillo** [reply]
Ok, I got confused because in the architectures, this is under the enterprise integration section, thanks.
